cmd/internal/ecs/add.go

```diff
@@ -840,15 +840,15 @@ func modifyTaskState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowS
 		Value: aws.String(akitaCreationTagValue),
 	})
 
-	apiKey, apiSecret := cfg.GetAPIKeyAndSecret()
+	pKey, pEnv := cfg.GetPostmanAPIKeyAndEnvironment()
 	input.ContainerDefinitions = append(input.ContainerDefinitions, types.ContainerDefinition{
-		Name: aws.String("akita-agent"),
+		Name: aws.String("postman-lc-agent"),
 		// TODO: Cpu and Memory should be omitted for Fargate; they take their default values for EC2 if omitted.
 		// For now we can leave the defaults in place, but they might be a bit large for EC2.
-		EntryPoint: []string{"/akita", "apidump", "--project", projectFlag},
+		EntryPoint: []string{"/postman-lc-agent", "apidump", "--collection", collectionId},
 		Environment: []types.KeyValuePair{
-			{Name: aws.String("AKITA_API_KEY_ID"), Value: &apiKey},
-			{Name: aws.String("AKITA_API_KEY_SECRET"), Value: &apiSecret},
+			{Name: aws.String("POSTMAN_API_KEY"), Value: &pKey},
+			{Name: aws.String("POSTMAN_ENV"), Value: &pEnv},
```

Comment:
pEnv may be blank -- in fact that is the expected case for production.

Reply:
Should we set this with `production` as default value ?